processing/src/main/java/org/apache/druid/query/CPUTimeMetricQueryRunner.java

```diff
@@ -78,16 +79,17 @@ public <RetType> RetType wrap(Supplier<RetType> sequenceProcessing)
               return sequenceProcessing.get();
             }
             finally {
-              cpuTimeAccumulator.addAndGet(JvmUtils.getCurrentThreadCpuTime() - start);
-              responseContext.addCpuNanos(JvmUtils.getCurrentThreadCpuTime() - start);
+              long cpuTimeDelta = JvmUtils.getCurrentThreadCpuTime() - start;
+              cpuTimeAccumulator.addAndGet(cpuTimeDelta);
+              responseContext.addCpuNanos(cpuTimeDelta);
             }
           }
 
           @Override
           public void after(boolean isDone, Throwable thrown)
           {
             if (report) {
-              final long cpuTimeNs = cpuTimeAccumulator.get();
+              final long cpuTimeNs = responseContext.getCpuNanos();
```

Comment:
This should be emitted from cpuTimeAccumulator because this emit is local to this QueryRunner

Reply:
cpuTimeAccumulator does not include Cpu time spent at Data nodes (when this code is run in the context of Broker). Earlier code emitted cpuTimeAccumulator.get() and that led to the bug that this does not include time spent at Data nodes (shown in the code below).

https://github.com/confluentinc/druid/blob/f0e71b5309a562e7f77653b3f8956b1f470dfd19/server/src/main/java/org/apache/druid/client/DirectDruidClient.java#L270